### PR TITLE
Avoid unsafe conditions for integration tests

### PR DESCRIPTION
**Pull request by brahmanim**
##### SUMMARY

Vyos Remove all Jinja templating expressions from asserts condition,  integration test fix.

##### ISSUE TYPE

- Test fix

##### ADDITIONAL INFORMATION

```paste below
https://docs.ansible.com/ansible/latest/porting_guides/porting_guide_9.html#playbook
```

### CODE REVIEW

**Review by Ruchip16 — APPROVED** [review]

**Comment by komaldesai13** [comment]
recheck

File: changelogs/fragments/integration_test_fix.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ---
 trivial:
-  - vyos_prefix_lists - fix vyos_prefix_lists in integration tests to resolve issues related to deletion.
+  - avoid_unsafe_conditions - Avoid unsafe conditions in integration tests to resolve issues related to assertion unsafe conditions.
```
Comment (KB-perByte) [inline changelogs/fragments/integration_test_fix.yml:3]: @brahmanim why are we updating an existing changelog entry?
Can we please add a new one?

**Review by KB-perByte — CHANGES_REQUESTED** [review]

File: changelogs/fragments/integration_test_fix.yml (hunk 1 above)
Comment (brahmanim) [reply changelogs/fragments/integration_test_fix.yml]: @KB-perByte I added a new avoid_unsafe_conditions.yml

**Comment by brahmanim** [comment]
recheck

**Review by KB-perByte — APPROVED** [review]

**Pull request merged**